lib/services/flowdock.ts

```diff
@@ -15,53 +15,239 @@ limitations under the License.
 */
 
 import * as Promise from 'bluebird';
+const Session = require('flowdock').Session;
```

Comment:
Don't use `const` to import legacy JS modules. If required, type them yourself (or create a file to be PRd to the DefinitelyTyped project). See the `import and Modules` section of the TS coding guide for more details.
The `typings/temp.d.ts` declaration file gives an example of us typing it ourselves for the functionality required.